### PR TITLE
Library screen & downloaded track screen UI

### PR DESCRIPTION
**Pull request by hails96**
# Pull request title
Library screen & downloaded track screen UI

# Redmine
https://edu-redmine.framgia.vn/issues/1994
https://edu-redmine.framgia.vn/issues/1995
https://edu-redmine.framgia.vn/issues/1997

### CODE REVIEW

**Comment by hails96** [comment]
ready

File: app/src/main/java/lsh/framgia/com/isoundcloud/screen/main/downloadedtrack/DownloadedTrackFragment.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    @Override
+    protected int getLayoutId() {
+        return R.layout.fragment_downloaded_track;
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getActivity() == null || getContext() == null) return;
+        ((MainActivity) getActivity()).setToolbarTitle(
```
Comment (DoanVanToan) [inline app/src/main/java/lsh/framgia/com/isoundcloud/screen/main/downloadedtrack/DownloadedTrackFragment.java]: https://developer.android.com/training/basics/fragments/communicating
em nên tạo 1 interface thay thì ép kiểu ntn 
và bắt sự kiện trong onAttach

File: app/src/main/java/lsh/framgia/com/isoundcloud/screen/main/downloadedtrack/DownloadedTrackFragment.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    @Override
+    public void onMenuClick(Track track) {
+
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        if (getActivity() == null || getContext() == null) return;
+        ((MainActivity) getActivity()).setToolbarTitle(
```
Comment (DoanVanToan) [inline app/src/main/java/lsh/framgia/com/isoundcloud/screen/main/downloadedtrack/DownloadedTrackFragment.java]: tương tự

**Comment by DoanVanToan** [comment]
commented

**Comment by hails96** [comment]
ready

File: app/src/main/res/layout/fragment_library.xml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                app:layout_constraintEnd_toStartOf="@id/text_playlist"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/text_playlist"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/dp_20"
+                android:text="Playlist"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_library.xml]: hardcode

**Pull request merged**